controller.js

```diff
@@ -48,4 +49,24 @@ async function getArticles (req, res, next) {
     }
 }
 
-module.exports = { getTopics, getEndpoints, getArticleById, getArticles };
\ No newline at end of file
+async function getCommentsByArticleId (req, res, next) {
+    const { article_id } = req.params;
+    const promises = [
+        selectCommentsByArticleId(article_id),
+        selectArticleById(article_id)
+    ]
+    Promise.all(promises).then((resolutions) => {
+        if (resolutions[0].length === 0) {
+            return Promise.reject({status: 404, msg: "No comments found"})
```

Comment:
we're doing a reject here based on no comments being found, but then we're also doing a reject for no comments found in the model?